src/contracts/Resources/config/doctrine.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Ibexa\DoctrineSchema\Database\DbPlatform\PostgreSqlDbPlatform;
+use Ibexa\DoctrineSchema\Database\DbPlatform\SqliteDbPlatform;
+use RuntimeException;
+
+return static function (ContainerConfigurator $container): void {
+    if (!isset($_ENV['DATABASE_URL'])) {
+        $_ENV['DATABASE_URL'] = 'sqlite://:memory:';
+    }
```

Comment:
This either doesn't work or I don't get the purpose. If I try to do a test which load schema, I get the following error
```
1) Ibexa\Tests\Integration\Test\Core\SampleTest::testCompilesSuccessfully
Symfony\Component\DependencyInjection\Exception\EnvNotFoundException: Environment variable not found: "DATABASE_URL".
```
and need to configure that env in phpunit.xml anyway.

Reply:
Fixed in 9233dc1d.

No idea why the environment superglobal has no effect here. I've read through the `Symfony\Component\DependencyInjection\EnvVarProcessor` which is supposed to handle this resolution (and it throws said excepton). We might investigate it further, but I feel that we should spend our time elsewhere for the time being.

Reply:
Well, actually I have found the reason why it worked for me.

It works only for the first time, when container compilation occurs. Afterwards, because this file is not executed, `$_ENV` is no longer filled with default value provided here.